response_operations_ui/templates/conversation-view.html

```diff
@@ -0,0 +1,52 @@
+{% extends "layouts/base.html" %}
+
+{% block page_title %} Create Message | Survey Data Collection {% endblock %}
+
+{% block main %}
+
+
+
+<h1 class="saturn" name="page-messages-title">{{ messages[0].get('subject') }}</h1>
+
+<div class="secure-message-conversation block">
+    <div class="secure-message-sent-message-meta">
+         <div class="secure-message--meta venus"><span class="sm-form-meta-label">Survey: </span><span id="survey">{{ messages[0].get('survey') }}</span></div>
+         <div class="secure-message--meta venus"><span class="sm-form-meta-label">RU Ref: </span><span id="ruref">    {{ messages[0].get('ruref') }}</span></div>
+         <div class="secure-message--meta venus"><span class="sm-form-meta-label">Business: </span><span id="ru">{{ messages[0].get('business') }}</span></div>
+         <div class="secure-message--recipient venus"><span class="sm-form-recipient-label">To: </span><span id="recipient">    {{ messages[0].get('to') }}  </span></div>
+
+    </div>
+    {% for msg in messages %}
```

Comment:
I don't think we should be shortening words, message is more readable than msg and is consistent with other uses e.g. https://github.com/ONSdigital/response-operations-ui/blob/3db04ba8f4c434d30c4ad88b0bb36a20ece144c2/response_operations_ui/templates/messages.html#L28

Reply:
I agree in isolation it's a preference thing, but it is inconsistent with other places in the codebase (where message in messages is used).